lib/python/vdm/server/HTTPListener.py

```diff
@@ -637,13 +758,21 @@ def post(database_id):
         if not inputs.validate():
             return jsonify(success=False, errors=inputs.errors)
 
+        arr = ["http-listener", "admin-listener", "internal-listener", "replication-listener", "zookeeper-listener", "client-listener"]
+        for servers in Global.SERVERS:
+            if servers['hostname'] == request.json['hostname']:
```

Comment:
We should use list comprehension here to filter out servers that don't have the same hostname as the hostname in the request and thus avoid this if.